sapphire/transformations/celestial.py

```diff
@@ -168,18 +184,22 @@ def equatorial_to_horizontal(latitude, longitude, timestamp, right_ascension,
 
     zenith, azimuth = horizontal_to_zenithazimuth(altitude, alt_azimuth)
 
-    return zenith, azimuth
+    return zenith, azimuth # NB: return is in zenithazimuth format not horizontal
```

Comment:
Please fix the function name or make it work as the function name suggests and actually return horizontal coordinates.